README.md

```diff
@@ -1,2 +1,243 @@
+
+### World-Class Meetup & Training Venue
+
+Part dwyl's of our mission to help everyone discover what they love doing requires us to share our knowledge through Meetups and training.
+
+We have _already **successfully**_ used the space for training both
+our own team members, our clients and complete strangers.
+
+Our plan is to create a _public_ training calendar for QuietSpace
+where specific training is regular.
+
+#### What Will We Learn?
+
+We already have a [52+](https://github.com/search?q=org%3Adwyl+learn)
+tutorials in various levels of "maturity" which we plan to convert into video content by recording a "complete beginners workshop"
+
+We have a _rough **order**_ for the learning material:
+https://github.com/dwyl/the-book/issues/69 but we expect to be "_dynamic_" and will respond to the requests/needs of the attendees/members.
+
+
+
+
+#### Training Registration System
+
+For the QuiteSpace-based training we will need a
+registration system/process so people can
+book a space on the training for a specific day/time.
+(_The **MVP** can use an
+**existing "booking" tool** provided we capture all the requirements
+before making a decision. as always we **prefer open source**._)
+
+
+
+
+### QuietSpace Timetable for Co-Working, Breaks and Meetups/Training
+
+The most productive people in the world all have a well-defined Timetable for getting their work done.
+
+|Start - End    | Description |
+|---------------|-------|
+| 04:00 - 06:00 | Healthy food preparation (_upstairs available for work, expect some noise will be made downstairs_) |
+| 06:00 - 07:00 | Self-guided Yoga / Pilates / Mindfulness (Downstairs), Reading or Quiet work Upstairs |
+| 07:00 - 10:00 | "Early Bird's" Focus Time (_absolute silence please this is when we get our best work done_) |
+| 10:00 - 10:30 | Morning break (_moderate chat/noise acceptable_) |
+| 10:30 - 12:30 | Morning Focus Time (_please whisper or use "instant messaging" to communicate_) |
+| 12:30 - 14:00 | Lunch Break (_moderate chat/noise acceptable though please be respectful of people working through lunch_) |
+| 14:00 - 16:00 | Afternoon Focus Time (_please whisper or use "instant messaging" to communicate_) |
+| 16:00 - 16:30 | Afternoon Break (_moderate chat/noise acceptable_) |
+| 16:30 - 18:30 | "Final Push" Focus Time (_please whisper or use "instant messaging" to communicate_) |
+| 18:30 - 19:00 | "Show & Tell", "Get Help", "Q&A", "Decompress Time" |
+| 19:00 - 20:30 | Meetup, Event or Evening Training (_training space bookable by members_) |
+| 20:30 - 21:00 | Tidy up the space ready for next session. |
+| 21:00 - 23:00 | "Late Shift" Focus Time (_Upstairs only. please whisper or use "instant messaging" to communicate_) |
+| 23:00 - 04:00 | "Night Owls Club" (_for people who work best at night. Upstairs only.  Please apply for a door key. Door must be kept locked except for entry/exit. No Slumber Parties! ;-) Last person out switch off all lights/heaters and set the alarm!_) |
+
```

Comment:
@markwilliamfirth agreed. 👍 
The _expectation_ I want to set is: to have "_best possible_ place to work". 🥇 
Right now QS is _definitely **not**_ the best place to get creative work done in London. 😞 

I'm happy for you or _anyone_ `else` to cut out anything from this readme/PR. ✂️ 
and add what ever you think is important/priority for making this thing work in 3 months. 📆 

as discussed, I didn't write this PR because I think I'm "_best_" suited for this task. 🙃 
I just wanted to get this ball rolling :shipit: 
so we can have sort out our office space both for dwyl and for QuietSpace Ltd. ❤️  ✅ 
I _know_ you will do a _great_ job of sorting QS. 🚀 
So I've "_un-watched_" the repo 🔕 
and will only reply when mentioned/assigned to an issue. 🤙 
Thank you for sorting this for us! 🦄